noir-projects/noir-contracts/contracts/nft_contract/src/test/transfer_to_private.nr

```diff
@@ -69,41 +61,11 @@ unconstrained fn transfer_to_private_to_self_transfer_not_prepared() {
     // Setup without account contracts. We are not using authwits here, so dummy accounts are enough
     let (env, nft_contract_address, _, _, token_id) = utils::setup_and_mint(/* with_account_contracts */ false);
 
-    // Transfer was not prepared so we can use random value for the commitment
-    let transfer_preparer_storage_slot_commitment = random();
+    // Transfer was not prepared so we can use random value for the hiding point slot
```

Comment:
This test is called `to_self` but that's not accurate - it's just that the transfer was not prepared.

Reply:
https://github.com/AztecProtocol/aztec-packages/pull/9150/commits/df2368f67d62779a310f5bdc65a7d022bffb169d